frame/assets/src/lib.rs

```diff
@@ -184,6 +194,8 @@ pub struct AssetDetails<
 	zombies: u32,
 	/// The total number of accounts.
 	accounts: u32,
+	/// Whether the asset is frozen for permissionless transfers.
```

Comment:
this change will break any existing deployments

Reply:
is it being used anywhere yet?

Are you asking me to write some migration code?

Reply:
don't think so, but worth noting in case someone reading knows of a deployment.